.eslintrc

```diff
@@ -42,8 +46,10 @@
     "accessor-pairs": [2, {"setWithoutGet": true, "getWithoutSet": false}],
     "array-bracket-spacing": [2, "never"],
     "array-callback-return": 2,
+    "arrow-parens": [2, "as-needed"],
```

Comment:
The prevailing style is to only use it `as-needed`. There's 287 eslint errors if set to `always` and 34 errors if set to `as-needed` as in the PR.